src/test/shell/bazel/remote/build_without_the_bytes_test.sh

```diff
@@ -726,6 +726,182 @@ EOF
     //a:foo || "Failed to build //a:foo"
 }
 
+function test_downloads_minimal_symlink_to_source_file() {
```

Comment:
Optional but please consider moving these tests to java.

Reply:
I will move them once we agree on the overall approach.

Reply:
Moved.